sprokit/src/bindings/python/modules/loaders.py

```diff
@@ -84,15 +84,26 @@ def _isPackage(self, path):
         return False
 
     def _findPluginFilePaths(self, namespace):
+        """
+        Searches for modules in `namespace` that are reachable from the paths
+        defined in the PYTHONPATH environment variable.
+        """
```

Comment:
Given that there wasn't anything there before, this is better. I'm not sure we have a standard, but I have been using restructured/sphinx style in other projects (e.g. SMQTK).